### ISSUE TITLE
[feature] create class Trainer to train the models

### ISSUE THREAD

**Issue by nunenuh** [lombokai/onepiece-classifier#7]
- The trainer class in the constructor has parameter input
   - max_epochs
   - learning rate, 
   - optimizer
   - loss function

- The trainer class method fit  with parameter Models and OnepieceImageDataLoader

**Comment by nunenuh** [comment]
it has been done with review
